### ISSUE TITLE
Closing an unused tab causes "Process Interrupted" in the used tab.

### ISSUE THREAD

**Issue by GalaxyTimeMachine** [comfyanonymous/ComfyUI#4014]
### Expected Behavior

I can run a workflow in one browser tab and be able to close another without the first tab being interrupted.

### Actual Behavior

If I have 2 ComfyUI browser tabs open, queue a prompt in one of them and close the other tab, the queued prompt is interrupted and must be restarted.

### Steps to Reproduce

Described in behaviour

### Debug Logs

```powershell
got prompt
[rgthree] Using rgthree's optimized recursive execution.
</s><s>The image is a close-up portrait of a young woman with long, wavy brunette hair. She is wearing a pink dress and has a pink flower in her hair. The woman is resting her chin on her hand and is looking directly at the camera with a serious expression. The background is blurred, but it appears to be a beach with rocks and water. The overall mood of the image is dreamy and ethereal.</s>
Offloading model...
INFO: the IPAdapter reference image is not a square, CLIPImageProcessor will resize and crop it at the center. If the main focus of the picture is not in the middle the result might not be what you are expecting.
Requested to load CLIPVisionModelProjection
Loading 1 new model
Requested to load SDXLClipModel
Loading 1 new model
Processing interrupted
Prompt executed in 54.98 seconds
```

### Other

Using Edge browser.

**Comment by GalaxyTimeMachine** [comment]
I should add that this doesn't happen every time, but frequently enough to be annoying.

**Comment by siddie** [comment]
Can confirm. Same behaviour. Chrome My case:  putting tasks into a queue via an API, from a different machine. 

When I decided to shutdown the ComfyUI tabs on the server machine, I saw the API client failed to recevied the workflow results. I checked the server console, saw the same "Processing interrupted".

**Comment by ltdrdata** [comment]
I cannot reproduce this.
Is it still happen without any custom node?

**Comment by dmakhrov** [comment]
I confirm this issue too. Brave browser [Version 1.73.104 Chromium: 131.0.6778.204 (Official Build) (64-bit)]
To reproduce this problem, you might need a lengthy process running like, for example, video generation. Then, go open the running server in another tab, load and reload workflows, close this tab, open a new one and so on. It will happen. Eventually.

**Comment by NullEqualsZero** [comment]
Shame here (not sure if it is tab-related) but it seems for me it happens with specific "heavy" workflows more often. Although I can't recreate it intentionally I am not sure what exactly causes it. I just now it happens sometimes... (sorry for not mbeing uch of a help)

**Comment by ltdrdata** [comment]
@huchenlei Is there a possibility that the workflow execution will be interrupted when the connection with the client is lost?

**Comment by Dawgmastah** [comment]
I can confirm this behavior, running on chrome, if I close another tab, I get processing interrupted on the other running tab.
It will interrupt one image, if there are more in the queue, it will continue.

**Comment by Dawgmastah** [comment]
To be clearer by tab I mean a browser tab, not one of the new(er) comfyui tabs inside the same browser tab.

(Thinking about it, this wasnt an issue before this feature as far as I recall.)

**Comment by zgauthier2000** [comment]
This is happening to me with the latest comfy. It kills the currently running workflow and switches to the next one in the queue any time the browser tab is closed (or even if i try to close it, get a warning dialog, and then hit cancel). This didn't happen before a recent update. If I kill Chrome's process rather than normally closing the tab, it doesn't break the running workflow. Surely this isn't by design.

**Comment by CB5K** [comment]
> This is happening to me with the latest comfy. It kills the currently running workflow and switches to the next one in the queue any time the browser tab is closed (or even if i try to close it, get a warning dialog, and then hit cancel). This didn't happen before a recent update. If I kill Chrome's process rather than normally closing the tab, it doesn't break the running workflow. Surely this isn't by design.

Yep same thing happening too me on opera GX. I have multiple comfy UI tabs open and closing one causes the current task to be cancelled with a "process interrupted"

**Comment by XuruiPro** [comment]
Is there a temporary repair plan available

**Comment by GalaxyTimeMachine** [comment]
> Is there a temporary repair plan available

If you use the tabs within the ComfyUI work area, instead of the browser, then this problem isn't there.

https://github.com/user-attachments/assets/9758116b-c024-4ccb-a29b-832fe184f88c

**Comment by papandadj** [comment]
up

**Comment by papandadj** [comment]
How come when I close the browser, the task automatically stops. Print Processing interrupted

**Comment by VladoPortos** [comment]
Still a problem :(

**Comment by XuruiPro** [comment]
It may be caused by the 'ComfyUI-Easy-Use' extension. Try disabling the 'ComfyUI-Easy-Use' extension

**Comment by yinghuan007** [comment]
> It may be caused by the 'ComfyUI-Easy-Use' extension. Try disabling the 'ComfyUI-Easy-Use' extension

yeah!!!!really because of this,thanks!!!

**Comment by astradamus** [comment]
> ### Expected Behavior
> 
> I can run a workflow in one browser tab and be able to close another without the first tab being interrupted.
> ### Actual Behavior
> 
> If I have 2 ComfyUI browser tabs open, queue a prompt in one of them and close the other tab, the queued prompt is interrupted and must be restarted.

To be clear, the expected behavior is actually that closing any tab doesn't interrupt anything, ever, period. The workflow is not "running in the tab" it's running in the server software, simply hiding the interface shouldn't have any effect at all.

Fortunately that's exactly how Comfy behaves when the Easy Use addon is deleted! I notified the creator of Easy Use, hopefully a fix soon.
